### DISCUSSION TITLE
Mosaico - Responsive Email Template Editor with Codeigniter Backend

### DISCUSSION THREAD

**Discussion by ar27111994** [voidlabs/mosaico#697] [Q&A]
Hi, I've successfully created a full featured **PHP Codeigniter** backend implementation for Mosaico. You can see a live demo at http://mosaicoci.ar27111994.com/. Login Username is `admin@admin.com` and Password is `password`. Whereas source code can be found at [https://github.com/ar27111994/Mosaico-CodeIgniter-Ion-Auth](https://github.com/ar27111994/Mosaico-CodeIgniter-Ion-Auth). It'd love this integration to be featured in Mosaico Wiki and other suitable places, representing the user base of Codeigniter PHP Framework, who want to integrate Mosaico in their Codeigniter Web Applications. Thanks.

**Comment by arthuralmeidap** [comment]
Hi @ar27111994, what about the client-side implementation? Do you have an example?

**Comment by ar27111994** [comment]
Hi, @arthuralmeidap, you can check out the whole working demo at http://mosaicoci.ar27111994.com/. Client side implementation is provided by Mosaico itself. I have just edited it's two main files and used Mosaico's plugin creation capabilities to plug-in my CodeIgniter backend implementation into Mosaico pre-built frontend.

**Comment by arthuralmeidap** [comment]
Is provided by Mosaico? I couldn't find this implementation. Can you show me?

**Comment by ar27111994** [comment]
Did you follow all the installation steps provided in the documentation? Did you install all the Node.js modules and packages and build them properly?
